### ISSUE TITLE
Separator should be lighter in color on Manage packages page.

### ISSUE THREAD

**Issue by anangaur** [NuGet/NuGetGallery#6137]
Manage packages page:
![image](https://user-images.githubusercontent.com/14800916/42235071-68ff9b88-7eab-11e8-98b5-cd7d2b31ca69.png)

**Expected**: like Manage organizations page:
![image](https://user-images.githubusercontent.com/14800916/42235091-7fbd1f30-7eab-11e8-8fbf-ee8fda160623.png)

**Comment by xavierdecoster** [comment]
Pretty sure that was verified as part of [this PR](https://github.com/NuGet/NuGetGallery/pull/6029/files#diff-0e18edf26cad58746f00981649dd91f3R5). 

Don't we regenerate the `.css` files at compile time when we make changes to the `.less` file? Or did I forget a manual step I didn't know about?

**Comment by loic-sharma** [comment]
Yes, there's a manual step to regenerate the `.css` files. It's hidden away in [this OneNote page](https://microsoft.sharepoint.com/teams/NuGet/_layouts/15/WopiFrame.aspx?sourcedoc={15b3fddd-11c7-4f67-8f84-456297b3b279}&action=edit&wdLOR=&wd=target%28Specs.one%7C910181c5-8695-4a1c-9792-a56dbb86f1c3%2FNew%20Page%20Instructions%20-%20CSS%20and%20LESS%7Cac44e307-8bf4-42db-8b11-edda8f809603%2F%29&wdorigin=703).

TLDR, in `src/Bootstrap`:
1. `npm install -g grunt-cli`
2. `npm install`
3. `grunt`

**Comment by xavierdecoster** [comment]
I ran those, but don't see any changes in my git repo when replacing those .css files. 
Also: the `src\NuGetGallery\Content\gallery\css\bootstrap.css` and `src\NuGetGallery\Content\gallery\css\bootstrap-theme.css` are explicitly excluded from the git repository?

**Comment by xavierdecoster** [comment]
I found that the CSS change [is in the `dev` branch](https://github.com/NuGet/NuGetGallery/blob/dev/src/Bootstrap/dist/css/bootstrap-theme.css#L1161), but [not in `master`](https://github.com/NuGet/NuGetGallery/blob/master/src/Bootstrap/dist/css/bootstrap-theme.css#L1156)?

Seems like a merge gone wrong?

**Comment by xavierdecoster** [comment]
Checking the diff between `dev` and `master` shows that this will be fixed in the next PR to `master`.

**Comment by skofman1** [comment]
@xavierdecoster , can I close this issue or would you like to keep it open for tracking the fix?

**Comment by xavierdecoster** [comment]
Assigned to self, will track and verify this fix on next deployment :)
